### PR TITLE
Adding A Blog named as Top 4 VS Code extension every developer should use

### PR DESCRIPTION
**Pull request by pulkitgovrani**
Please review this

### CODE REVIEW

**Review by BaseMax — APPROVED** [review]
LGTM.

**Comment by BaseMax** [comment]
A suggestion: It would be great if you link to add-ins page.

**Review by Majalian — DISMISSED** [review]

File: doc/source/vs-code-extension-every-developer-should-use.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+It does the work everytime whenever someone saves the work in VS code and therefore reduces the time of formatting. It is very useful whenever a team is working on some project as it leads to better and clear understanding of the code.
+
+## 3. Javascript Code Snippets:
+
+As the name suggests this extension gives you many javascript code snippets thus reducing time to setup the code. It provides different types of ES6 Snippets including arrays, DOM manipulation, etc.
+
+## 4. Bracket Pair Colorizer:
+
+This is one of the personal favourite extensions of mine. It includes simple functioning i.e, it maches starting and ending brackets in vs code which eventually helps you to resolve the errors faster.
+
```
Comment (johnbampton) [inline doc/source/vs-code-extension-every-developer-should-use.md:26]:
```suggestion
```

File: doc/source/vs-code-extension-every-developer-should-use.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# :boom:Top 4 VS Code extensions every developer should use
```
Comment (johnbampton) [inline doc/source/vs-code-extension-every-developer-should-use.md:1]:
```suggestion
# :boom: Top 4 VS Code extensions every developer should use
```

**Review by johnbampton — CHANGES_REQUESTED** [review]

**Review by johnbampton — CHANGES_REQUESTED** [review]
Please add your menu link in `index.rst` in the general technical area.

`doc/source/index.rst`

**Comment by pulkitgovrani** [comment]
> Please add your menu link in `index.rst` in the general technical area.
> 
> `doc/source/index.rst`

I have done that, Please review it again

File: doc/source/what-programming-language.md [hunk 3]
```diff
@@ -13,10 +13,12 @@ programming language. Let’s look at the below example for understanding why
 python is easier to learn.
```
Comment (jbampton) [inline doc/source/what-programming-language.md:13]:
```suggestion
Python is easier to learn.
```

File: doc/source/what-programming-language.md [hunk 4]
```diff
@@ -13,10 +13,12 @@ programming language. Let’s look at the below example for understanding why
...
-can print “Hello World” in Python:
+![C++](_static/images/What_Programming_Language_Images/c++.png) 
+This is how we can print “Hello World” in Java:
+
+![Java](_static/images/What_Programming_Language_Images/java.png)
+This is how we can print “Hello World” in Python:
+
 ![Python](_static/images/What_Programming_Language_Images/python.png)
 
 So I am pretty sure you must have made your mind till now that learning python
```
Comment (jbampton) [inline doc/source/what-programming-language.md:24]:
```suggestion
So I am pretty sure you must have made your mind till now that learning Python
```

**Review by jbampton — CHANGES_REQUESTED** [review]

File: doc/source/vs-code-extension-every-developer-should-use.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+This is an extension which makes a local server enabling the user to see all the characterstics of the app. It works perfectly for both dynamic and static web pages.
+We can open the live server by right clicking on the mouse while VS Code is running behind and then selecting the Open with Live Server option.
+
+## 2. Prettier:
+
+Prettier is a code formatting extension therefore it helps you to clean and format the code automatically.
+It does the work everytime whenever someone saves the work in VS code and therefore reduces the time of formatting. It is very useful whenever a team is working on some project as it leads to better and clear understanding of the code.
+
+## 3. Javascript Code Snippets:
```
Comment (jbampton) [inline doc/source/vs-code-extension-every-developer-should-use.md:19]:
```suggestion
## 3. JavaScript Code Snippets:
```

File: doc/source/vs-code-extension-every-developer-should-use.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+As the name suggests this extension gives you many javascript code snippets thus reducing time to setup the code. It provides different types of ES6 Snippets including arrays, DOM manipulation, etc.
```
Comment (jbampton) [inline doc/source/vs-code-extension-every-developer-should-use.md:21]:
```suggestion
As the name suggests this extension gives you many JavaScript code snippets thus reducing time to setup the code. It provides different types of ES6 Snippets including arrays, DOM manipulation, etc.
```

File: doc/source/vs-code-extension-every-developer-should-use.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+## 4. Bracket Pair Colorizer:
+
+This is one of the personal favourite extensions of mine. It includes simple functioning i.e, it maches starting and ending brackets in vs code which eventually helps you to resolve the errors faster.
```
Comment (jbampton) [inline doc/source/vs-code-extension-every-developer-should-use.md:25]:
```suggestion
This is one of the personal favourite extensions of mine. It includes simple functioning i.e, it matches starting and ending brackets in vs code which eventually helps you to resolve the errors faster.
```

**Review by jbampton — CHANGES_REQUESTED** [review]

File: doc/source/vs-code-extension-every-developer-should-use.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
+# :boom: Top 4 VS Code extensions every developer should use
+
+## What is VS Code?
+
+Visual studio code is a text editor which is open source i.e, free to use and it is made by microsoft.
+VS Code’s simplicity and easy to use features makes itself favourite choice for the developers. It has a feature through which you can install the extensions according to your needs. Therefore these extensions are also open source and are made by the developers for the community.  
+Here are some of the extensions which are going to increase your productivity as a developer:
+
+##  1. Live Server:
```
Comment (jbampton) [inline doc/source/vs-code-extension-every-developer-should-use.md:9]:
```suggestion
## 1. Live Server:
```

**Review by jbampton — CHANGES_REQUESTED** [review]

**Pull request closed without merging**